lacci/lib/shoes/errors.rb

```diff
@@ -16,23 +16,23 @@ class NoDrawablesFoundError < Shoes::Error; end
 
   class NoSuchStyleError < Shoes::Error; end
 
-  class NoLinkableIdError < Shoes::Error; end
+  class NoSuchLinkableIdError < Shoes::Error; end
 
   class BadLinkableIdError < Shoes::Error; end
 
-  class UnregisteredShoesEvent < Shoes::Error; end
+  class UnregisteredShoesError < Shoes::Error; end
```

Comment:
In this case it's for an unregistered Shoes event. So we'd like to keep the word Event but add Error. Same for DoubleRegistered below.

Reply:
Ok  working on it.

Reply:
@noahgibbs  is it good now? or need further changes